### PR TITLE
Added new test for upload files

### PR DESCRIPTION
**Pull request by mshahrukh2**
Includes Android and Web tests for uploading file

### CODE REVIEW

File: src/test/java/com/znsio/sample/e2e/businessLayer/theapp/FileUploadBL.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import com.znsio.e2e.runner.Runner;
+import com.znsio.sample.e2e.entities.SAMPLE_TEST_CONTEXT;
+import com.znsio.sample.e2e.screen.theapp.FileUploadScreen;
+import org.apache.log4j.Logger;
+import org.assertj.core.api.SoftAssertions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FileUploadBL {
+    private static final Logger LOGGER = Logger.getLogger(AppBL.class.getName());
```
Comment (vishal07111998) [inline src/test/java/com/znsio/sample/e2e/businessLayer/theapp/FileUploadBL.java:14]: Logger name not matching

File: src/test/java/com/znsio/sample/e2e/screen/theapp/FileUploadScreen.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                .getId());
+        Platform platform = Runner.fetchPlatform(Thread.currentThread()
+                .getId());
+        LOGGER.info(SCREEN_NAME + ": Driver type: " + driver.getType() + ": Platform: " + platform);
+        Visual visually = fetchEyes(Thread.currentThread()
+                .getId());
+
+        switch(platform) {
+            case android:
+              //  return new FileUploadScreenAndroid(driver, visually);
```
Comment (vishal07111998) [inline src/test/java/com/znsio/sample/e2e/screen/theapp/FileUploadScreen.java:29]: commented code can be removed

Comment (mshahrukh2) [reply src/test/java/com/znsio/sample/e2e/screen/theapp/FileUploadScreen.java]: Updated

File: src/test/java/com/znsio/sample/e2e/businessLayer/theapp/FileUploadBL.java (hunk 1 above)
Comment (mshahrukh2) [reply src/test/java/com/znsio/sample/e2e/businessLayer/theapp/FileUploadBL.java]: Updated.

**Comment by mshahrukh2** [comment]
Will be merge with PR #142

**Pull request closed without merging**